### DISCUSSION TITLE
Welcome to SoftwareChatter Discussions!

### DISCUSSION THREAD

**Discussion by seabbs** [cmmid/SoftwareChatter#2] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by TimTaylor** [comment]
Cheers for starting this @seabbs. I'm Tim, currently working as an RSE within LSHTM.  Always happy to hear software related chatter of all kinds but currently have a particular interest in how we can make software in LSHTM more sustainable and also reproducible in the longer term.

**Reply by seabbs** [reply]
👋 I agree both really critical issues at the moment.
